arviz/plots/distplot.py

```diff
@@ -90,6 +90,8 @@ def plot_dist(
         Keywords passed to ax.pcolormesh. Ignored for 1D KDE.
     hist_kwargs : dict
         Keywords passed to the histogram.
+    is_circular : str
```

Comment:
Based on numpydoc style I'd recommend:

```
is_circular : {None, "radians", "degrees"}, optional
```

or 

```
is_circular : {None, "radians", "degrees"}, default None
```

(maybe changing None with False depending on what you prefer based on comment above)

Reply:
Also, should `True` be added to the set of options too?